dynamic_instrumentation/Cargo.toml

```diff
@@ -10,6 +10,7 @@ c2rust-analysis-rt = { path = "../analysis/runtime"}
 cargo = "0.62"
 cargo-util = "0.1"
 indexmap = "1.8"
+itertools = "0.10"
```

Comment:
I added `itertools` because it greatly simplifies the `args` debug printing, and it's already used in the workspace.